code/modules/xenomorph/xeno_structures.dm

```diff
@@ -1050,3 +1050,39 @@ TUNNEL
 	light_initial_color = LIGHT_COLOR_PURPLE
 	ammo = /datum/ammo/xeno/sticky
 	firerate = 5
+
+
+/obj/structure/xeno/evotower
```

Comment:
a range check with other evo towers maybe? Also, marines should have a way of finding those, wether with the pool tracker, or it emitting sound, or it emitting light

Reply:
done